app/assets/javascripts/repository_navigation.js

```diff
@@ -34,7 +34,7 @@
       branch = $('#revision-branch-select'),
       selects = tag.add(branch),
       branch_selected = branch.length > 0 && revision.val() == branch.val(),
-      tag_selected = tag.length > 0 && revision.val() == tag.getValue();
+      tag_selected = tag.length > 0 && revision.val() == tag.val();
```

Comment:
Identifier 'tag_selected' is not in camel case.<br>Expected '===' and instead saw '=='.